resolver/resolver.go

```diff
@@ -311,15 +287,9 @@ func (res *Resolver) HandleNonMesos(w dns.ResponseWriter, r *dns.Msg) {
 		// set refused
 		m.SetRcode(r, 5)
 	} else {
-
-		proto := "udp"
-		if _, ok := w.RemoteAddr().(*net.TCPAddr); ok {
```

Comment:
not sure why this was done this way. maybe someone on the ops team has some insight. i'm a little leery of changing behavior like this without a good reason.